controllers/rhmi/rhmi_controller.go

```diff
@@ -1271,6 +1273,26 @@ func (r *RHMIReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return nil
 }
 
+func (r *RHMIReconciler) isHiveLabelPresent(client k8sclient.Client) (bool, error) {
+	namespace, err := resources.GetWatchNamespace()
+	if err != nil {
+		return false, err
+	}
+
+	ns, err := resources.GetNS(context.TODO(), namespace, client)
+	if err != nil {
+		return false, fmt.Errorf("could not retrieve %s namespace:", err)
+	}
+
+	for label := range ns.ObjectMeta.Labels {
```

Comment:
Is it _ns.ObjectMeta.Labels_ or ns.Labels?
Or maybe doesn't matter?

Reply:
Thanks Brian, good point